trollsift/parser.py

```diff
@@ -304,7 +305,7 @@ def format_spec_to_regex(field_name, format_spec):
                 width=width,
                 precision=precision
             )
-        if ftype == 's' and align and align.endswith('='):
+        if ftype in ['s', ''] and align and align.endswith('='):
```

Comment:
```suggestion
        if ftype in ('s', '') and align and align.endswith('='):
```

I think I'd prefer a tuple here (although I realize earlier in the code is also not a tuple (that could be changed too).